packages/yew-router/src/utils.rs

```diff
@@ -0,0 +1,62 @@
+use std::collections::HashMap;
+use wasm_bindgen::{JsCast, JsValue};
+
+pub fn base_url() -> Option<String> {
+    match yew::utils::document().query_selector("base") {
+        Ok(Some(base)) => {
+            let base = base
+                .unchecked_into::<web_sys::Element>()
+                .attributes()
+                .get_named_item("href")
+                .expect("base without href")
```

Comment:
I did it because, afaik, `base` without `href` is invalid. I'm not sure if we should allow that. `trunk` inserts `/` as href when nothing is specified.

Reply:
As far as I know, `base` without `href` isn't valid. Should we allow that situation?

Reply:
That's understandable. I'll make it return `None` if no href is found